frontend/resourceadm/utils/resourceUtils/resourceUtils.test.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  languageStringMock,
+  missingInputLanguageStringTestMock,
+  nbLanguageMock,
+  resourcePageTextfieldInputMock1,
+  resourcePageTextfieldInputMock2,
+  resourcePageTextfieldInputMock3,
+  resourcePageTextfieldInputMock4,
+  resourcePageTextfieldInputMock5,
+  resourcePageTextfieldInputMock6,
+  resourceTypeOptionDefaultMock,
+  resourceTypeOptionIncorrectMock
+} from "resourceadm/data-mocks/resourceMocks";
+import { convertResourceTypeToDisplayString, getMissingInputLanguageString, getResourcePageTextfieldError, mapLanguageKeyToLanguageText } from "./resourceUtils";
+
+describe('getResourcePageTextfieldError', () => {
+  it('returns false when the field have valid data', () => {
+    const hasError: boolean = getResourcePageTextfieldError(resourcePageTextfieldInputMock1);
+    expect(hasError).toBeFalsy();
+  });
+
+  it('returns true when the a language field is empty', () => {
+    const hasErrorMissingNB: boolean = getResourcePageTextfieldError(resourcePageTextfieldInputMock2);
+    const hasErrorMissingNN: boolean = getResourcePageTextfieldError(resourcePageTextfieldInputMock3);
+    const hasErrorMissingEN: boolean = getResourcePageTextfieldError(resourcePageTextfieldInputMock4);
```

Comment:
It's quite challenging to distinguish the mocks apart from their names like 1, 2, and 3. I recommend inserting the data directly into the test.

The following example illustrates a unit test using mock data, showcasing how we can conveniently modify it between test cases while maintaining excellent readability.
```suggestion
   const defaultMock = { nb: '', nn: '', en: '' }

    const hasErrorMissingNB: boolean = getResourcePageTextfieldError({...defaultMock, nb: 'Valid' });
    const hasErrorMissingNN: boolean = getResourcePageTextfieldError({...defaultMock, nn: 'Valid' });
    const hasErrorMissingEN: boolean = getResourcePageTextfieldError({...defaultMock, en: 'Valid' });
```